config/templates/app_geoserver_audit/app_geoserver_audit.json

```diff
@@ -1,7 +1,7 @@
 {
   "measurementType": "historic",
   "context": {
-    "!inputDb": "{{ '/tmp/fluent-bit-logs.db' if outputLocalLambdaEnabled else '/logs/fluent-bit-logs.db' }}",
+    "!inputDb": "{{ '/tmp/fluent-bit-logs.db' if outputLocalLambdaEnabled else '/apps_data/agents/fluent-bit/fluent-bit-logs.db' }}",
```

Comment:
Would be the original. inputDbFile is overwritten by the defaults for our setup. This is a generic path for containers.